metaci/testresults/tests/test_robot_importer.py

```diff
@@ -514,3 +514,26 @@ def test_gus_bus_payload_metadata():
             "key1": True,
             "key2": "hello, world",
         }
+
+
+@responses.activate
+@pytest.mark.django_db
+def test_gus_bus_bad_payload_metadata():
+    """Verify that all build-specific metadata is added to the payload"""
+    flowtask = FlowTaskFactory()
+    flowtask.build_flow.build.org = OrgFactory()
+    flowtask.build_flow.build.repo.metadata = {"key1": True, "key2": "hello, world"}
+    responses.add(
+        "POST",
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/test-results/",
```

Comment:
The route in gus-bus is defined without a trailing slash (https://github.com/SFDO-Tooling/gus-bus/blob/main/gus_bus/main.py#L244); we should be sure that MetaCI calls it without the slash or there's an extra redirect.

Reply:
E           requests.exceptions.ConnectionError: Connection refused by Responses - the call doesn't match any registered mock.
E           
E           Request: 
E           - POST https://webhook/test-results/
E           
E           Available matches:
E           - POST https://webhook/test-results URL does not match

is happening when I remove it from the tests @davisagli

Reply:
@jofsky you also need to change it here https://github.com/SFDO-Tooling/MetaCI/blob/main/metaci/testresults/robot_importer.py#L333